### PR TITLE
Rebased and updated version of PR145 (robot_status msg publishing)

### PR DESCRIPTION
**Pull request by gavanderhoorn**
As per subject.

All the hard work done by @yamokosk and @keerthanamanivannan in #145.

### CODE REVIEW

**Comment by gavanderhoorn** [comment]
Sigh GH<->Travis.

**Comment by gavanderhoorn** [comment]
Let's wait for Travis and then merge.

**Comment by gavanderhoorn** [comment]
And of course a review from someone else, as this is my own PR ..

**Comment by gavanderhoorn** [comment]
Note: this should *not* be squash-merged.

File: abb_driver/rapid/ROS_stateServer.mod [hunk 1]
```diff
@@ -77,13 +77,14 @@ ERROR
     RAISE;  ! raise errors to calling code
 ENDPROC
 
-! signalRobotEStop : System Output
-! signalMotorOn : System Output
+! signalExecutionError : System Output
+! signalExecutionError : System Output
```
Comment (keerthanamanivannan) [inline abb_driver/rapid/ROS_stateServer.mod]: @gavanderhoorn I know that this is just a commented line, but why do we have two of this signalExecutionError?

Comment (gavanderhoorn) [reply abb_driver/rapid/ROS_stateServer.mod]: Because I messed up the rebase. That is all. I'll fix it.

Comment (gavanderhoorn) [reply abb_driver/rapid/ROS_stateServer.mod]: Good catch. Should be fixed.

**Comment by gavanderhoorn** [comment]
@ros-industrial/ros-i-reviewers: could any of you take a look here?

**Comment by gavanderhoorn** [comment]
I would like to get this in so we can push a new release of `ros-industrial/abb`.

**Review by Levi-Armstrong — APPROVED** [review]

**Comment by gavanderhoorn** [comment]
Thanks @Levi-Armstrong.

Did you have "rebase and merge" selected? That should not have been necessary.

**Comment by gavanderhoorn** [comment]
And a big thanks to @keerthanamanivannan and @yamokosk for all the work on this PR. I only did some management and testing.

**Pull request merged**